.github/workflows/reviews.yml

```diff
@@ -0,0 +1,92 @@
+name: Reviews
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/reviews.yml
+  pull_request_target:
+    types: [ready_for_review]
+  workflow_call:
+
+permissions: {}
+
+jobs:
+  tag:
+    name: Request
+    runs-on: ubuntu-24.04-arm
+    steps:
+      - name: Check out the PR at the base commit
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          path: trusted
+          sparse-checkout: ci
+
+      - name: Install Nix
+        uses: cachix/install-nix-action@526118121621777ccd86f79b04685a9319637641 # v31
+        with:
+          extra_nix_config: sandbox = true
+
+      - name: Build the requestReviews derivation
+        run: nix-build trusted/ci -A requestReviews
+
+      # See ./codeowners-v2.yml, reuse the same App because we need the same permissions
+      # Can't use the token received from permissions above, because it can't get enough permissions
+      - uses: actions/create-github-app-token@df432ceedc7162793a195dd1713ff69aefc7379e # v2.0.6
+        if: vars.OWNER_APP_ID
+        id: app-token
+        with:
+          app-id: ${{ vars.OWNER_APP_ID }}
+          private-key: ${{ secrets.OWNER_APP_PRIVATE_KEY }}
+          permission-administration: read
+          permission-members: read
+          permission-pull-requests: write
+
+      - name: Wait for comparison to be done
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        with:
+          script: |
+            const run_id = (await github.rest.actions.listWorkflowRuns({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              workflow_id: 'eval.yml',
+              event: context.eventName,
+              head_sha: context.payload.pull_request.head.sha
+            })).data.workflow_runs[0].id
```

Comment:
Neat! So even when this wasn't triggered by `workflow_call` it'll still find the latest `eval.yml` job

At the cost of adding some complexity, would it be worth it to pass `run_id` as an optional `input` to the workflow, so that it doesn't need to be queried when this workflow _was_ triggered directly by `eval.yml`?

```yml
workflow_call:
  inputs:
    eval_id:
      type: number
      required: false
```

```js
const run_id = context.inputs.eval_id || (await ...).data.workflow_runs[0].id
```